pkg/actors/actors.go

```diff
@@ -1199,7 +1199,7 @@ func (a *actorsRuntime) executeTimer(actorType, actorID, name, dueTime, period,
 	req := invokev1.NewInvokeMethodRequest(fmt.Sprintf("timer/%s", name))
 	req.WithActor(actorType, actorID)
 	req.WithRawData(b, invokev1.JSONContentType)
-	_, err = a.callLocalActor(context.Background(), req)
+	_, err = a.Call(context.Background(), req)
```

Comment:
Please revert back to `callLocalActor` since timers can only be created for actors that are activated in the local Dapr sidecar.

Reply:
Ok, I see this is the only change. I think we should close this PR, as mentioned above, actor timers can only be activated locally.